### ISSUE TITLE
Proposal: django-invitations

### ISSUE THREAD

**Issue by brylie** [jazzband/help#237]
# Project Proposal

Please fill in the details below to confirm you adhere to the [Jazzband Guidelines](https://jazzband.co/about/guidelines), and also add the package name to the issue title.

## Project Details
django-invitations

## Project Description
Generic invitations solution with adaptable backend and support for django-allauth. All emails and messages are fully customisable.

The original developer has expressed that they can no longer maintain the project. However, there are several open pull requests from community contributors.

## Project Leads
Please list anyone who you would like to be made a project lead.

- @brylie

## Confirmation of Guidelines

Please confirm the following:

- [ ] I am the project owner, or have permission from the project owner, to request a transfer.

- [ ] This project is a [viable candidate](https://jazzband.co/about/guidelines#viability) for Jazzband.

- [ ] This project fulfills the [documentation requirements](https://jazzband.co/about/guidelines#documentation).

- [ ] This project is [tested](https://jazzband.co/about/guidelines#tests).

- [ ] I am happy to adopt Jazzband's [code of conduct](https://jazzband.co/about/guidelines#conduct) in this project.

- [ ] I am happy to include Jazzband's [contributing guidelines](https://jazzband.co/about/guidelines#contributing-guidelines) in this project.

**Comment by brylie** [comment]
Hei all. Can we please review this proposal? It has heart support but no reviewers.

**Comment by MrCordeiro** [comment]
@brylie, what should be the next steps here? Is there any Jazzband member we should contact?

**Comment by jezdez** [comment]
I'm sorry this is stuck on the proposal backlog, I think the things that jump out that needs to be checked before a transfer can happen:

- the docs are sparse and would need to be ported to a Sphinx project and hosted on Read The Docs. This can be done after the transfer but I need acknowledgement that this would happen
- the package metadata doesn't contain data which license this is released under, while the LICENSE file is GPL3
- the CI setup would need to be ported to GitHub Actions, but virtually all other Jazzband projects can be used as inspiration

**Comment by MrCordeiro** [comment]
Thanks, @jezdez!

@brylie, I see you have a [PR](https://github.com/bee-keeper/django-invitations/pull/166) going. Need any help? I could try adding to it.

**Comment by brylie** [comment]
Thanks, @MrCordeiro. Feel free to contribute to the PR in any way that would help the proposal move forward :-)

**Comment by MrCordeiro** [comment]
> * the CI setup would need to be ported to GitHub Actions, but virtually all other Jazzband projects can be used as inspiration

@jezdez, I was looking at the other projects and got a question. Should I just create a Test workflow for now and then the Release workflow when the project is moved, or do I create both now? Thanks!

**Comment by jezdez** [comment]
> > * the CI setup would need to be ported to GitHub Actions, but virtually all other Jazzband projects can be used as inspiration
> 
> @jezdez, I was looking at the other projects and got a question. Should I just create a Test workflow for now and then the Release workflow when the project is moved, or do I create both now? Thanks!

You can create both of them and just change the names in the release workflow, it'll be identical for this project and just needs the release key to enable it, that I'd do.

**Comment by MrCordeiro** [comment]
[Done](github.com/bee-keeper/django-invitations/pull/166), @jezdez!
The only thing missing is to port the docs to a Sphinx project, be we can do this after the transfer, right?

**Comment by jezdez** [comment]
@MrCordeiro Yep, totally fine. Feel free to transfer the repo with the usual repo transfer functionality, a ticket to track the rest of the TODOs will be opened automatically.

**Comment by MrCordeiro** [comment]
@brylie, have you had any news of the transfer?

**Comment by brylie** [comment]
@MrCordeiro, nope. I think @bee-keeper will need to initiate the transfer from their account.

**Comment by valberg** [comment]
I guess that only @bee-keeper can do the transfer? Is forking the project a viable alternative if we do not hear from them?

**Comment by bee-keeper** [comment]
Hi, sorry i've been slow on this.  I just made the transfer now so should be all good?

**Comment by valberg** [comment]
@bee-keeper Awsome! No need to be sorry - we all have stuff to do that gets in the way of other stuff :)

I can see django-invitations on the jazzband website now: https://jazzband.co/projects/django-invitations 👏

**Comment by MrCordeiro** [comment]
@bee-keeper You'll see that there is a checklist of things required to complete the transfer [here](https://github.com/jazzband/django-invitations/issues/172), like changing the user for coveralls and adding the Jazzband bot as a project maintainer on Pypi, that only you'll be able to do.

**Comment by chrysle** [comment]
@jezdez Since the transfer has been completed, this can be closed.
